resources/qgis_global_settings.ini

```diff
@@ -45,6 +45,9 @@ connections-xyz\OpenStreetMap\username=
 connections-xyz\OpenStreetMap\zmax=19
 connections-xyz\OpenStreetMap\zmin=0
 
+# The image format selected by default when adding a WMS
+WMSDefaultFormat="image/png"
```

Comment:
Shouldn't this be empty so that we default to the current behaviour of using the first format?

Reply:
Right, done!